presto-hive/src/main/java/io/prestosql/plugin/hive/security/LegacyAccessControl.java

```diff
@@ -110,7 +111,7 @@ public void checkCanDropTable(ConnectorSecurityContext context, SchemaTableName
         }
 
         if (!context.getIdentity().getUser().equals(target.get().getOwner())) {
-            denyDropTable(tableName.toString(), "Owner of the table is different from session user");
+            denyDropTable(tableName.toString(), format("Owner of the table ('%s') is different from session user ('%s')", target.get().getOwner(), context.getIdentity().getUser()));
```

Comment:
Do we leak owner name with this kind of message?

Reply:
Yes, this is the goal of the change.

We e.g. propagate exceptions from HDFS which reveal current user, file owner and other information.

Also note there seems to be no command in Presto I could use to know the table owner, so I currently need to _guess_ the owner to be able to be able to drop it.